openedx/core/djangoapps/programs/utils.py

```diff
@@ -130,6 +130,14 @@ def get_program_marketing_url(programs_config, mobile_only=False):
 
     return marketing_url
 
+def get_program_subscriptions_marketing_url():
```

Comment:
I think we need to put a check in place here to restrict this url on mobile view. AFAIK we are not showing subscription pages on mobile view hence it makes sense that we don't send this URL as well. You can take a look at `b2c_subscriptions_enabled` for reference

Reply:
sure